### ISSUE TITLE
Add database repair/deletion options

### ISSUE THREAD

**Issue by Mik-S-UMS** [UniversalMediaServer/UniversalMediaServer#4979]
It is not uncommon for users to have database problems after upgrading and they find they are unable to find any content on their renderers. Just had another one [here](https://www.universalmediaserver.com/forum/viewtopic.php?t=16143).

I have noticed in the logs that it mentions you can use a repair tool to recover it but I have no idea what this is or how to use it, so the only solution I know is to delete the database so a new one is made.

`
net.pms.database.DatabaseEmbedded Database migration failed: File corrupted while reading record: "C:/ProgramData/UMS/database/medias.mv.db". Possible solution: use the recovery tool [90030-214]
`

I think there should be some notification or something obvious in the settings that shows if there is a problem connecting to the database and gives the option to backup, restore, delete, or repair it without needing to manually doing it.

**Comment by SubJunk** [comment]
Yeah this is a good idea. I'm disappointed to see how often this is happening. It also makes me wonder whether h2database is the right tool for us. We don't do a lot of technical things with it
